packages/ui/src/AccessControlSection.svelte

```diff
@@ -36,7 +36,7 @@
       <span class="ml-1">
         <ToggleRadio bind:value={access} defaultValue="ownable" disabled={required} />
       </span>
-      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts/4.x/api/access">
+      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts/5.x-rc/api/access">
```

Comment:
This link will redirect to the latest version:

```suggestion
      <HelpTooltip align="right" link="https://docs.openzeppelin.com/contracts/api/access">
```